src/test/java/com/devonfw/ide/sonarqube/common/impl/config/ConfigurationMapperTest.java

```diff
@@ -21,8 +26,7 @@
       "{\"name\":\"component1\",\"dependencies\":[\"component2\"]}," + //
       "{\"name\":\"component2\",\"dependencies\":[\"component3\"],\"nonTransitiveDependencies\":[\"io.oasp.module.jpa\"]},"
       + //
-      "{\"name\":\"component3\",\"dependencies\":[]}" +
-      "]}}";
+      "{\"name\":\"component3\",\"dependencies\":[]}" + "]}}";
```

Comment:
The idea of these `//` at the end of the lines was to prevent Eclipse formatter from merging the lines and to make the JSON somewhat formatted and therefore more readable.
After the `//` was removed, now the formatter joined the two lines and the `+` does not seem to make sense anymore.
We we can not yet use multiline string literals (introduced in Java14 with `"""` syntax), it is better to keep aligning the JSON lines with the `//` workaround.

Reply:
yes, sure, corrected.